app/Application/CreateWallet/CreateWalletService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Application\CreateWallet;
+
+use App\Application\CryptoDataStorage\CryptoDataStorage;
+use Exception;
+
+class CreateWalletService
+{
+    /**
+     * @var CryptoDataStorage
+     */
+    private CryptoDataStorage $cryptoDataStorage;
+
+    /**
+     * CoinStatusService constructor.
+     * @param CryptoDataStorage $cryptoDataStorage
+     */
+    public function __construct(CryptoDataStorage $cryptoDataStorage)
+    {
+        $this->cryptoDataStorage = $cryptoDataStorage;
+    }
+
+    /**
+     * @return string
+     * @throws Exception
+     */
+    public function createWallet(): string
+    {
+        $user_wallet = $this->cryptoDataStorage->createWallet();
```

Comment:
Consider changing one of the methods name

Reply:
method in CreateWalletService could be called just execute()